package.json

```diff
@@ -32,37 +32,14 @@
     "get-pkg-repo": "4.1.1",
     "hosted-git-info": "^2.1.4"
   },
-  "dependencies": {
-    "@babel/runtime": "^7.14.8",
-    "@pixi/app": "^7.1.1",
-    "@pixi/constants": "^7.1.1",
-    "@pixi/core": "^7.1.1",
-    "@pixi/display": "^7.1.1",
-    "@pixi/extensions": "^7.1.1",
-    "@pixi/graphics": "^7.1.1",
-    "@pixi/math": "^7.1.1",
-    "@pixi/mesh": "^7.1.1",
-    "@pixi/mesh-extras": "^7.1.1",
-    "@pixi/particle-container": "^7.1.1",
-    "@pixi/sprite": "^7.1.1",
-    "@pixi/sprite-animated": "^7.1.1",
-    "@pixi/sprite-tiling": "^7.1.1",
-    "@pixi/text": "^7.1.1",
-    "@pixi/text-bitmap": "^7.1.1",
-    "@pixi/ticker": "^7.1.1",
-    "@react-spring/animated": "^9.6.1",
-    "@react-spring/types": "^9.6.1",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-spring": "^9.6.1"
-  },
```

Comment:
While I don't entirely love either way of doing it, I'd deliberately added these shared packages to the root so it was easier to keep track of versioning across the whole monorepo, rather than specifying potentially separate versions in each package

Reply:
My reasoning for this was that if we have a v6 component package and eventually a v8 package we would need to have individual packages defining there pixi versions?

Reply:
Yes, good point 👍